### DISCUSSION TITLE
Weather Widget 2 does not work

### DISCUSSION THREAD

**Discussion by bouleusis** [blackadderkate/weather-widget-2#153] [General]
Hello
The widget does not work and gives me an error message:

```
file:///home/javier/.local/share/plasma/plasmoids/org.kde.weatherWidget-2/contents/ui/main
.qml:120:5: Type OpenWeatherMap unavailable

file:///home/javier/.local/share/plasma/plasmoids/org.kde.weatherWidget-2/contents/ui/prov
iders/OpenWeatherMap.qml:18:1: module "QtQuick.XmlListModel" is not installed
```

I'm using KDE Plasma 5.27.7
Garuda Linux (Arch based) last update

I have been looking for QtQuick.XmlListModel module but have not found it.

Thanks in advance.

**Comment by blackadderkate** [comment]
I think it might be called qt5-xmlpatterns?

**Comment by skipppdev** [comment]
Apparently, installing `qml-module-qtquick-xmllistmodel` and rebooting seems to solve this issue.

**Reply by Satlinker** [reply]
Doesn't work for Kubuntu 24.10, Plasma 6.1.5
qml-module-qtquick-xmllistmodel ist schon die neueste Version (5.15.15-1)
qml-module-qtquick-xmllistmodel is already the newest version
